python/lsst/sdm_schemas/schemas/dp02_dc2.yaml

```diff
@@ -103,4890 +135,6840 @@ tables:
     description: GaaP flux uncertainty with optimal aperture after multiplying the seeing by 1.15. Forced on u-band.
     fits:tunit: nJy
+    tap:principal: 0
+    tap:column_index: 1195
   - name: u_gaapOptimalFlux_flag_bigPsf
     "@id": "#Object.u_gaapOptimalFlux_flag_bigPsf"
     datatype: boolean
     description: The Gaussianized PSF is bigger than the aperture. Forced on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1196
   - name: u_gaapPsfFlux
     "@id": "#Object.u_gaapPsfFlux"
     datatype: double
     description: GaaP flux with PSF aperture after multiplying the seeing by 1.15. Forced on u-band.
     fits:tunit: nJy
+    tap:principal: 0
+    tap:column_index: 1197
   - name: u_gaapPsfFluxErr
     "@id": "#Object.u_gaapPsfFluxErr"
     datatype: double
     description: GaaP flux uncertainty with PSF aperture after multiplying the seeing by 1.15. Forced on u-band.
     fits:tunit: nJy
+    tap:principal: 0
+    tap:column_index: 1198
   - name: u_hsmShapeRegauss_e1
     "@id": "#Object.u_hsmShapeRegauss_e1"
     datatype: double
     description: PSF-corrected shear using Hirata & Seljak (2003) regaussianization. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1199
   - name: u_hsmShapeRegauss_e2
     "@id": "#Object.u_hsmShapeRegauss_e2"
     datatype: double
     description: PSF-corrected shear using Hirata & Seljak (2003) regaussianization. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1200
   - name: u_hsmShapeRegauss_flag
     "@id": "#Object.u_hsmShapeRegauss_flag"
     datatype: boolean
     description: General failure flag, set if anything went wrong. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1201
   - name: u_hsmShapeRegauss_sigma
     "@id": "#Object.u_hsmShapeRegauss_sigma"
     datatype: double
     description: PSF-corrected shear using Hirata & Seljak (2003) regaussianization. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1202
   - name: u_iDebiasedPSF_flag
     "@id": "#Object.u_iDebiasedPSF_flag"
     datatype: boolean
     description: General failure flag, set if anything went wrong. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1203
   - name: u_iPSF_flag
     "@id": "#Object.u_iPSF_flag"
     datatype: boolean
     description: General failure flag, set if anything went wrong. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1204
   - name: u_iRound_flag
     "@id": "#Object.u_iRound_flag"
     datatype: double
     description: HSM moments. Measured on u-band.
     fits:tunit: pixel**2
+    tap:principal: 0
+    tap:column_index: 1205
   - name: u_i_flag
     "@id": "#Object.u_i_flag"
     datatype: boolean
     description: General failure flag, set if anything went wrong. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1206
```

Comment:
I'm pretty sure this is a flag column for `u_ixx`, etc., and should go right next to it (maybe going right next to it will mitigate the fact that the name is incredibly vague).